src/app/views/provisioning/confirm.controller.coffee

```diff
@@ -60,12 +60,14 @@ angular.module 'mnoEnterpriseAngular'
       ).finally(-> vm.isLoading = false)
 
     vm.orderEditable = () ->
+      return true if vm.subscription.product?.custom_schema
```

Comment:
Can we move the custom schema comment and add a comment for the skipPricingPlans.